### ISSUE TITLE
way to change ssl cipher suite ?

### ISSUE THREAD

**Issue by comsyspro** [kanaka/noVNC#516]
Hi,

when connecting to the webserver with "https://" the chrome browser shows the message "used old cipher suite" and it uses "TLS_RSA_128bit" encryption. How is it possible to use more modern cipher suites with stronger >256bit encryption? My ssl settings have very secure settings but it seems that this has no effect on the novnc webserver. Where in which file you should change something to be able to use much stronger ciphers?

Thanks.

**Comment by kanaka** [comment]
This is actually a websockify issue, not a noVNC issue. I believe it's already covered by this recent pull request: https://github.com/kanaka/websockify/pull/190 It's just waiting on some fallback code I believe.
